src/main/resources/templates/db_scripts/views/receiving_history_view.sql

```diff
@@ -29,7 +29,7 @@ SELECT pieces.id AS id,
        json_build_object(
            'id', pieces.jsonb ->>'id',
            'acqUnitIds', purchase_order.jsonb ->>'acqUnitIds',
```

Comment:
we missed to to chage this view in scope of UXPROD-4242, can you please add this change:
'acqUnitIds', title.jsonb ->>'acqUnitIds',
So we need to add one for left join with Title table

Reply:
By the way, can do in scope of separate PR

Reply:
The field was not used so removed it